tools/failures.py

```diff
@@ -194,7 +80,15 @@ def format_in_table(active_jobs, master):
             output += '\t'
             if test in data or '' in data:
                 try:
-                    output += tbplnames[test]['code']
+                    found = False
+                    for job in running_jobs:
+                        if job['name'] == test:
+                            output += job['job_type_symbol']
+                            found = True
+                            break
+
+                    if not found:
+                        raise Exception("unknown test: %s" % test)
                 except KeyError:
```

Comment:
The try..catch block can be removed since there will no longer be a KeyError as we are shifting to the 'found' variable approach.